test/specs/string-checker.js

```diff
@@ -357,6 +357,32 @@ describe('modules/string-checker', function() {
         });
     });
 
+    describe('throwing rules', function() {
+        beforeEach(function() {
+            checker = new StringChecker();
+            // register rule that throw
+            checker.registerRule({
+                configure : function() {},
+                getOptionName: function() { return 'thrower'; },
+                check : function() {
+                    throw Error('Here we are!');
+                }
+            });
+            checker.configure({
+                thrower: true
+            });
+        });
+
+        it('should be handled internally', function() {
+            var errs = checker.checkString('var a');
+            assert.equal(errs.getErrorCount(), 1);
+
+            var err = errs.getErrorList()[0];
+            assert.equal(err.rule, 'thrower');
+            assert.equal(err.message, 'Error: Here we are!');
```

Comment:
If we are going to alert the user to the error here, I'd think we should get more specific.

> Error running rule 'thrower'. This is an issue with JSCS and not your codebase. 

and then if in verbose, provide the actual error information and stack trace.